src/edwards.rs

```diff
@@ -822,7 +871,7 @@ impl ConstantTimeEq for AffinePoint {
 
 impl PartialEq for AffinePoint {
     fn eq(&self, other: &Self) -> bool {
-        bool::from(self.ct_eq(&other))
+        self.ct_eq(&other).unwrap_u8() == 1u8
     }
 }
```

Comment:
The projective here is perfect, is there a means of hyperlinking or somehow directing users from this code to worked examples in other parts of the library.